packages/frontend/src/components/accounts/RecoverAccountSeedPhrase.js

```diff
@@ -127,6 +152,21 @@ class RecoverAccountSeedPhrase extends Component {
                         handleChange={this.handleChange}
                     />
                 </form>
+                {showCouldNotFindAccountModal && (
+                    <CouldNotFindAccountModal
+                        onClickImport={async () => {
+                            const { secretKey } = parseSeedPhrase(seedPhrase);
+                            const recoveryKeyPair = KeyPair.fromString(secretKey);
+                            const implicitAccountId = Buffer.from(recoveryKeyPair.publicKey.data).toString('hex');
+                            await wallet.importZeroBalanceAccount(implicitAccountId, recoveryKeyPair);
```

Comment:
What's the likelihood of this failing? Do we need to catch and show an alert?

Reply:
Small but worth handling the error :)